### ISSUE TITLE
heap profiling visualization

### ISSUE THREAD

**Issue by Slach** [conprof/conprof#30]
i like idea about continuous profing
i hope you can implement heap profiling visualization
like a stacket time-series data like this
![image](https://user-images.githubusercontent.com/105560/68297974-4f695e80-00a9-11ea-9ed4-3a2842f2c1c4.png)
but more detailed to each type of allocation and maybe with link to source code

**Comment by brancz** [comment]
This is already available via Prometheus metrics. What are you thinking conprof to do here?

**Comment by Slach** [comment]
I mean something that can visualize 
```go tool pprof heapdata.gz``
not only as "top-K text-mode" console

yes it can be implemented over Prometheus metrics

but we will need too much metrics 
like
contprof_heap{host="XXX", allocation_type="module.Type", allocation_place="source_file.go:XX"}

**Comment by brancz** [comment]
Yes we don’t want to reinvent the wheel but we’d love to expose this data via a prometheus remote read endpoint so it can be queried from Prometheus.

**Comment by Slach** [comment]
Ah, ok I understood ;)

if you will expose this data as Prometheus metrics and get an opportunity to querying over it 

I will happily make some Prometheus dashboards for it
